### PR TITLE
Adam branch commits. 

### PR DESCRIPTION
**Pull request by creatinglake**

### CODE REVIEW

**Comment by creatinglake** [comment]
Not sure about my title of the pull request but hopefully that worked.

File: docs/network/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ roles:
   repository. Community Group decisions are made by consensus and sent to the
   Board of Governors for review and
   voting. [Join the Community Group here.](https://www.w3.org/community/veres-one/)
-* **Advisory Committees** are comprised of area experts that advise the Board
+* **Advisory Council** is comprised of area experts that advise the Board
```
Comment (dlongley) [inline docs/network/index.md]: This should say "The" before advisory council.

File: docs/network/governance.md [hunk 2]
```diff
@@ -65,7 +65,13 @@ entities such as Accelerators who utilize the Veres One Blockchain to provide
 services to customers. Community Group decisions are made by consensus and sent
 to the Board of Governors for review and voting.
 
-## Board of Governors Checks and Balances
+Since the Veres One Community Group is consensus based Community Group
+participants are checked by the other participants in the Community Group.
+There is also a final check on Community Group decisions by the Board of
+Governors who accept or reject Community Group decisions based on whether they
+comply with the Veres One Mission.
```
Comment (dlongley) [inline docs/network/governance.md]: I think we should strike this whole paragraph. We've already said decisions are made by consensus above and that the output of the CG goes to the Board of Governors [for a final check]. The only new content is a note about how decisions are made -- which is whether or not the Board of Governors thinks that the consensus decision complies with the Veres One Mission. If that's necessary to say we should just add another sentence to the top paragraph:

"The Board of Governors will vote to accept or reject Community Group decisions based on their compliance with the Veres One Mission."

File: docs/network/governors.md [hunk 3]
```diff
@@ -32,14 +32,14 @@ the proposal. If they determine that the proposal does conflict with the Mission
...
-All non-time sensitive decisions made by the Board of Directors must be made
+All non-time sensitive decisions made by the Board of Governors must be made
 by consensus which is defined as general agreement with no principled
 objections.
 
 Time sensitive decisions may be made by a 2/3rds majority vote, but only after
 exhausting a consensus-based approach.
 
-All decisions made must be publicly recorded and announced.
+All decisions must be publicly recorded and announced.
```
Comment (dlongley) [inline docs/network/governors.md]: Nit: Switching the order to "announced and recorded" sounds better.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: docs/network/governance.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ unilateral fashion.
 
 All parties are held accountable by legal agreements.
 
-## Node Checks and Balances
+## Nodes
```
Comment (msporny) [inline docs/network/governance.md]: -1 to removing "Checks and Balances"... this entire page is about checks and balances. Suggest rewording to: "Checks and Balances for Nodes".

File: docs/network/governance.md (hunk 2 above)
Comment (msporny) [reply docs/network/governance.md]: Yes, agreed, favor @dlongley's wording.

File: docs/network/governors.md (hunk 3 above)
Comment (msporny) [reply docs/network/governors.md]: +1

**Review by msporny — CHANGES_REQUESTED** [review]

File: docs/network/governance.md (hunk 4 above)
Comment (creatinglake) [reply docs/network/governance.md]: Not all of the copy is about checks and balances. This page defines the role of the entity AND then describes how that entity is accountable to and can be checked by the others. The repetition of "checks and balances seems unnecessarily redundant to me.  A compromise might be the format. "Nodes, Checks and Balances".

File: docs/network/governance.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ unilateral fashion.
 
 All parties are held accountable by legal agreements.
 
-## Node Checks and Balances
+## Nodes, Checks and Balances
```
Comment (dlongley) [inline docs/network/governance.md]: If this is the desired title style, it should use a colon `:` instead of a comma `,`. (Same all the way down for the test of the titles).

**Review by dlongley — CHANGES_REQUESTED** [review]

File: docs/network/governance.md (hunk 5 above)
Comment (creatinglake) [reply docs/network/governance.md]: +1, thanks.

Comment (dlongley) [reply docs/network/governance.md]: Oh, I just read your comment. Hmm, I need to rethink this, it's messier than it needs to be.

Comment (dlongley) [reply docs/network/governance.md]: Ok, well, we can leave it with the colon for now then! :)

Comment (msporny) [reply docs/network/governance.md]: @creatinglake -- I've given my inpu, you make the call. Don't water it down or overcomplicate it on my account.

**Review by msporny — APPROVED** [review]
